demos/list.html

```diff
@@ -486,10 +485,10 @@ <h3>Example - Avatar with Text and icon</h3>
                 </a>
               </li>
               <li class="mdc-list-item">
-                <img class="mdc-list-item__start-detail grey-bg"
+                <img class="mdc-list-item__start-detail"
                      src="/images/animal2.svg"
-                     width="56" height="56" alt="Sleuth"></span>
-                Sleuth
+                     width="56" height="56" alt="Sloth">
```

Comment:
FYI I think this was also hit in another PR? Did that get merged already?

Reply:
Yes, that already got merged, but they missed the trailing `</span>`

Reply:
BTW, I only noticed the invalid markup because IntelliJ is awesome and automatically highlights stuff like that